src/renderers/webgl/WebGLBackground.js

```diff
@@ -72,6 +72,7 @@ function WebGLBackground( renderer, state, geometries, premultipliedAlpha ) {
 			}
 
 			boxMesh.material.uniforms.tCube.value = background;
+			boxMesh.material.map = background;
```

Comment:
This is a bit hacky... I think we need to devise a better solution... Not sure what that would be at the moment.

Reply:
Agreed.